sources/core/objects/Grille.cpp

```diff
@@ -37,15 +37,36 @@ void Grille::AttackBy(IdPtr<Item> item)
             SetPassable(D_ALL, Passable::FULL);
             cutted_ = true;
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
+            return;
         }
         else
         {
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
             Delete();
+            return;
         }
     }
+    Breakable::AttackBy(item);
```

Comment:
It is possible to break `Grille` in SS13 with bare hands. I think, it is reasonable to add such feature here.